routing/road_graph.hpp

```diff
@@ -312,6 +314,8 @@ class IRoadGraph : public RoadGraphBase
     }
   }
 
+  /// \note |m_fakeIngoingEdges| and |m_fakeOutgoingEdges| are maps of sorted vectors.
```

Comment:
s/are maps of/map junctions to/

That is, they are not maps of vectors, they are maps whose values are sorted vectors.

Reply:
Fixed.